### PR TITLE
Fixed the CSS on Officer Page 

### PR DESCRIPTION
**Pull request by seemavora**
Condensed the code and made sure the navbar is not affected by the officer cards.

### CODE REVIEW

File: src/Pages/TheTeam/TheTeam.js [hunk 1]
```diff
@@ -1,27 +1,14 @@
...
-// import Seema from './teamPics/2019-2020/seema.png'
-// import Alex from './teamPics/2019-2020/alex.png'
-// import Alex2 from './teamPics/2019-2020/alex1.png'
-// import Thai from './teamPics/2019-2020/thai.png'
-// import Surabhi from './teamPics/2019-2020/serbi.png'
-// import Nanar from './teamPics/2019-2020/nanar.png'
+import OfficerCard from './OfficerCard'
 
-export default class TheTeam extends React.Component {
+export default class Example extends React.Component {
```
Comment (evanugarte) [inline src/Pages/TheTeam/TheTeam.js]: make the class name TheTeam

File: test/frontend/OfficerCard.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  })
+
+  it('Should render the name of the officers', () => {
+    expect(wrapper.find('.name-title').get(0).props.children).to.equal(
+      officerInfo.name
+    )
+  })
+
+  it('Should render the major of the officers', () => {
+    expect(wrapper.find('.major-title').get(0).props.children).to.equal(
```
Comment (evanugarte) [inline test/frontend/OfficerCard.test.js]: great job on this!!!

File: src/Pages/TheTeam/the-team.css [hunk 3]
```diff
@@ -1,11 +1,3 @@
-.Page{
-  padding-top: 20px;
-  display: inline-block;
-}
-
-.Panel{
-  padding-left: 17%;
-}
 
 .Title {
```
Comment (evanugarte) [inline src/Pages/TheTeam/the-team.css]: can you change the class name Title to title?

File: src/Pages/TheTeam/the-team.css [hunk 4]
```diff
@@ -17,16 +9,8 @@
...
-  margin-left: 12%;
-}
-
-#tab1 {
+ #tab1 {
   margin: auto;
   padding-top: 5px;
   margin-left: -6%;
-}
+}
```
Comment (evanugarte) [inline src/Pages/TheTeam/the-team.css]: remove trailing whitespace on this line.

File: test/frontend/OfficerCrad.test.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+/* global describe it */
```
Comment (evanugarte) [inline test/frontend/OfficerCrad.test.js]: can you remove this file? it seems to be a duplicate

**Comment by seemavora** [comment]
Screen Shot 2020-01-20 at 1 31 39 PM

Navbar now extends to the full page.

**Comment by ThaiQ** [comment]
Normal to Full size screen
image

Small size:
image

File: test/officerManager.js [hunk 6]
```diff
@@ -103,8 +103,7 @@ describe('OfficerManager', () => {
       const form = {
         name: 'pinkUnicorn',
         email: 'test@test.com',
-        level: 2,
-        team: 'dev',
+        role: 'dev',
```
Comment (evanugarte) [inline test/officerManager.js]: can you use the enums here?

File: test/officerManager.js [hunk 7]
```diff
@@ -125,8 +124,7 @@ describe('OfficerManager', () => {
...
```
Comment (evanugarte) [inline test/officerManager.js]: can you use the enums here?

File: test/officerManager.js [hunk 8]
```diff
@@ -148,8 +146,7 @@ describe('OfficerManager', () => {
...
```
Comment (evanugarte) [inline test/officerManager.js]: can you use the enums here?

File: test/officerManager.js [hunk 9]
```diff
@@ -219,8 +216,7 @@ describe('OfficerManager', () => {
           expect(response).to.have.length(1)
           expect(response[0].email).to.be.eql('test@test.com')
           expect(response[0].name).to.be.eql('pinkUnicorn')
-          expect(response[0].level).to.be.eql(2)
-          expect(response[0].team).to.be.eql('dev')
+          expect(response[0].role).to.be.eql('dev')
```
Comment (evanugarte) [inline test/officerManager.js]:
```suggestion
          expect(response[0].role).to.be.eql(form.role)
```

File: test/officerManager.js [hunk 10]
```diff
@@ -219,8 +216,7 @@ describe('OfficerManager', () => {
           expect(response).to.have.length(1)
           expect(response[0].email).to.be.eql('test@test.com')
```
Comment (evanugarte) [inline test/officerManager.js]:
```suggestion
          expect(response[0].email).to.be.eql(form.email)
```

File: test/officerManager.js [hunk 11]
```diff
@@ -219,8 +216,7 @@ describe('OfficerManager', () => {
...
           expect(response[0].name).to.be.eql('pinkUnicorn')
```
Comment (evanugarte) [inline test/officerManager.js]:
```suggestion
          expect(response[0].name).to.be.eql(form.name)
```

File: test/officerManager.js [hunk 12]
```diff
@@ -219,8 +216,7 @@ describe('OfficerManager', () => {
...
-          expect(response[0].level).to.be.eql(2)
-          expect(response[0].team).to.be.eql('dev')
+          expect(response[0].role).to.be.eql('dev')
           expect(response[0].major).to.be.eql('major')
```
Comment (evanugarte) [inline test/officerManager.js]:
```suggestion
          expect(response[0].major).to.be.eql(form.major)
```

File: test/officerManager.js (hunk 9 above)
Comment (evanugarte) [reply test/officerManager.js]: do this for all the other test cases too. we shouldnt be hardcoding the test cases

File: test/officerManager.js (hunk 6 above)
Comment (ThaiQ) [reply test/officerManager.js]: yes this is a mock data, just there for placeholder

File: api/models/OfficerManager.js [hunk 13]
```diff
@@ -20,17 +20,11 @@ const OfficerSchema = new Schema(
     instagram: {
```
Comment (ThaiQ) [inline api/models/OfficerManager.js]: Change this to ```linkedIn```

**Pull request closed without merging**